internal/projection/client.go

```
@@ -370,10 +392,63 @@ func projectionRequest(userID uint64) map[string]string {
+// not-yet-projected user (tier 3). A negative result is a valid cached entry,
+// not an error.
+func (c *Client) loadFetch(ctx context.Context, userID uint64, lname string) (fetchEntry, error) {
+	if view, found, projected, err := c.store.GetFetch(ctx, userID, lname); err == nil && projected {
+		return fetchEntry{fetch: view, found: found}, nil
+	}
+
+	reply, err := bus.RequestJSONTimeout[struct {
+		Fetches []FetchView `json:"fetches"`
+	}](ctx, c.nc, c.subjects.Fetches, projectionRequest(userID), c.rpcTimeout)
+	if err != nil {
+		return fetchEntry{found: false}, nil
```

golangci-lint (2.12.2)

[error] error is not nil (line 426) but it returns nil

(nilerr)